google/cloud/spanner_dbapi/cursor.py

```diff
@@ -198,8 +202,12 @@ def executemany(self, operation, seq_of_params):
         """
         self._raise_if_closed()
 
+        self._many_result_set = StreamedManyResultSets()
```

Comment:
I don't know that you need a separate class for this, you could [chain the `self._itr` calls](https://docs.python.org/3/library/itertools.html#itertools.chain.from_iterable) instead to do the same thing.

It may still be useful to have a class to e.g. get the length of the combined result set (although it looks like `StreamedResultSet` itself doesn't have a length), but I don't think you need it for what it's doing now.